README.md

```diff
@@ -31,6 +31,27 @@ bin/start-dependencies
 npm run start:dev
 ```
 
+### Initial setup (Apple silicon setup)
+
+```
+docker-compose pull
+
+(nvm use)
+npm install
+
+Pull down following repositories:
+hmpps-assess-risks-and-needs, community-api, hmpps-auth, and offender-assessments-api
+
+Add '127.0.0.1 hmpps-auth' to your /etc/hosts file
```

Comment:
This is already mentioned above, I wonder if it's worth adding 'M1' sections in-between the 'general' sections? (Duplication is harder to maintain)

Reply:
Good point! I've changed the layout to make it clearer and to remove duplication